tools/import-wraps.py

```diff
@@ -137,22 +138,20 @@ def get_provide(wrap):
     deps = [i for i in deps if i]
     return progs, deps
 
-def add_to_db(wrap, versions, releases):
-    releases.setdefault(wrap, {})
-    releases[wrap].setdefault('versions', [])
-    releases[wrap].setdefault('dependency_names', [])
-    releases[wrap].setdefault('program_names', [])
-    versions = [Version(f'{version}-{revision}') for version, revision in versions]
-    versions = sorted(versions, reverse=True)
-    versions = [v._s for v in versions]
+def add_to_db(wrap: str, versions: T.List[T.Tuple[str, int]], releases: T.Dict[str, configparser.SectionProxy]):
+    releases.setdefault(wrap, {}) # type: ignore
```

Comment:
`releases: T.Dict[str, configparser.SectionProxy]`

But then ***every single usage site*** is breaking that rule and getting ignored. So I don't quite understand the point of it all. What do we gain from this typing anyway?

Reply:
What types fits it better? I don't understand this function as I called out in the description.

Reply:
I don't know what it is either, but the obvious answer seems to be a `T.Any` rather than `configparser.SectionProxy` and then ignoring it every single time.

Moreso given that nowhere is configparser used for the releases variable.

Reply:
Okay, so the general layout of releases is:

```
releases = {
    'foo': {
        versions: ['2.0, '1.0'],
        'program_names': ['foo-config'],
        'dependency_names': ['libfoo']
    }
}
```

So intuitively, this would be:
```
T.Dict[str, T.Dict[str, T.List[str]]]
```

And indeed, that works after removing all ignores.